.github/workflows/deploy.yaml

```diff
@@ -462,6 +462,40 @@ jobs:
           REGISTRATION_SERVICE_VERSION: 0.0.1-SNAPSHOT
           DATASPACE_DID_HOST: ${{ needs.Deploy-Dataspace.outputs.dataspace_did_host }}
 
+      - name: 'Wait for participant registration'
+        run: |               
+          retryCount=0
```

Comment:
why is this script duplicated (with entrypoint.sh) rather than extracted?

Reply:
I have consolidated the duplicated code.